client/components/gravatar/index.jsx

```diff
@@ -64,10 +73,18 @@ module.exports = React.createClass( {
 		}
 
 		const alt = this.props.alt || this.props.user.display_name;
-		const avatarURL = this.getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
+
+		const avatarURL = this.props.tempImage ||
+			this.getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
```

Comment:
> Could we extract these props out of props

Yes :) I extracted the props in `render` and also in `getResizedImageURL`.

> shouldn't we write a new selector for getting the user's avatar_URL

Do you mean a new redux selector? If so, some components use their own user objects, not relying on the users in `state.users`, so they won't be able to pass in an ID and let `<Gravatar />` get the selected user's info. For example, `ReaderAvatar`.